### PR TITLE
add logging to eligible cases service

### PR DESCRIPTION
**Pull request by jamiestephenkerr**

### CODE REVIEW

File: services/eligible_case_service.py [hunk 1]
```diff
@@ -1,16 +1,83 @@
+import logging
 from typing import List
 from models.questionnaire_case_model import QuestionnaireCaseModel
+from cloud_functions.logging import setup_logger
+
+setup_logger()
```
Comment (tomphp) [inline services/eligible_case_service.py:6]: This shouldn't be here, it should only get called once during start up, not in included files.

**Review by tomphp — CHANGES_REQUESTED** [review]

File: services/eligible_case_service.py (hunk 1 above)
Comment (jamiestephenkerr) [reply services/eligible_case_service.py]: Removed and tests still pass

**Review by jamiestephenkerr — COMMENTED** [review]
Ok

**Review by JAWilliamsONS — APPROVED** [review]
lgtm

**Pull request merged**